src/mmw/sass/components/_buttons.scss

```diff
@@ -86,7 +86,6 @@
 }
 
 .btn-secondary {
-    height: $height-md;
```

Comment:
Consider targeting this further for the button on this page. Removing it from this generic spot has effects elsewhere in the app. For example, see the "Details" and "New Project" buttons in Modeling view (diff highlighted in purple):

![diff](https://user-images.githubusercontent.com/1430060/30652185-b9018886-9df5-11e7-9115-6b90ceba2413.png)

Reply:
How'd you make this diff?

Reply:
I took two screenshots with identical browser sizes, called `original.png` and `changed.png`. Then I used [GraphicsMagick](http://www.graphicsmagick.org/) with this command to generate `diff.png`:

```bash
$ gm compare -highlight-style assign -highlight-color purple -file diff.png changed.png original.png
```

You can also use [ImageMagick](https://www.imagemagick.org/), which is the original and more standard, but GraphicsMagick has better hardware acceleration support on macOS I think.